[📝 Doc] Update hidden form fields list and postgres version recommandations (#4846)
GeotrekCE/Geotrek-admin: docs/advanced-configuration/interface.rst

Hideable form fields
^^^^^^^^^^^^^^^^^^^^^

.. warning::

>    There may be other exceptions, for example if a **Suricate workflow** is enabled.

Reviewer: ```suggestion
   There are exceptions, for example if a **Suricate workflow** is enabled.
```

   Be careful with the *recommended* fields required to publish an object 
   (``COMPLETENESS_FIELDS`` and ``COMPLETENESS_LEVEL`` settings).  
   They will indeed be hidden, but this can raise an **error** or a **warning** when saving.

>    To summarize, a field can be hidden only if it meets **all** the following conditions:

Reviewer: Je mettrais cette règle avant le warning qui mentionne ses exceptions.

```suggestion
   A field can be hidden only if it meets **all** the following conditions:
```

   - It is included in the form's ``fields`` list  
   - It is **not required**  

> 

Reviewer: Je pense qu'il serait bien de mentionner également le fait que le champ ne doit pas être présent dans la liste `not_hideable_fields` du formulaire, car même si pour l'instant cette liste n'est utilisée que dans un seul formulaire, c'est une option qui est présente pour tous les forms et son utilisation pourrait se généraliser dans le futur.

Author: Ok, merci pour tes retours, pris en compte dans les derniers commits.
